securedrop/translations/messages.pot

```diff
@@ -902,13 +902,16 @@ msgstr ""
 msgid "Click the <img src=\"{icon}\" alt=\"&quot;Security Level&quot; button\"> in the toolbar above"
 msgstr ""
 
-msgid "Click <b>Change</b> to open Security Level preferences"
+msgid "Click <b>Settings</b> to open Security Level preferences"
 msgstr ""
 
-msgid "Select <b>Safest</b> and close the preferences tab"
+msgid "Click <b>Change...</b> and select <b>Safest</b> "
```

Comment:
NB: trailing whitespace (I think Weblate will trim it, but just in case)

Reply:
(removed during .pot regeneration)